hedy_translation.py

```diff
@@ -28,8 +28,9 @@ def translate_keywords(input_string, from_lang="en", to_lang="nl", level=1):
     punctuation_symbols = ['!', '?', '.']
 
     keyword_dict = keywords_to_dict(to_lang)
-    if level > 7:
```

Comment:
Why are we removing the level check here? processing blocks only needs to be done for levels over 7

Reply:
I added the check that was already present in hedy.py, the preprocess_input() function. That one also only uses preprocess_blocks when level > 7